h2o-docs/src/booklets/v2_2015/source/sw/sections/api.tex

```diff
@@ -3,27 +3,18 @@ \section{Programming API}
 \subsection{Starting H2O Services}
 
 \begin{lstlisting}[style=Scala]
-val sc: SparkContext = ...
-val hc = H2OContext.getOrCreate(sc)
+import org.apache.spark.h2o._
+val hc = new H2OContext(getOrCreate(spark)}
 \end{lstlisting} 
-or:
-\begin{lstlisting}[style=Scala]
-val sc: SparkContext = ...
-val hc = new H2OContext(sc).start()
-\end{lstlisting}
 
 When the number of Spark nodes is known, it can be specified in the \texttt{getOrCreate} call:
 
 \begin{lstlisting}[style=Scala]
-val hc = H2OContext.getOrCreate(sc, numOfSparkNodes)
+val hc = H2OContext.getOrCreate(spark, numOfSparkNodes)
 \end{lstlisting}
-or, in start method of \texttt{H2OContext}:
 
-\begin{lstlisting}[style=Scala]
-val hc = new H2OContext(sc).start(numOfSparkNodes)
-\end{lstlisting}
 
-The former variant is preferred, because it initiates and starts \texttt{H2OContext} in one call and can be used to obtain already existing \texttt{H2OContext}. It is semantically the same as the latter variant though.
+This initiates and starts \texttt{H2OContext} in one call and can be used to obtain the already existing \texttt{H2OContext}. 
 
 \subsection{Memory Allocation}
 
```

Comment:
The paragraph bellow saying that H2O resides in the same executor JVM as Spark applies only to internal sparkling water backend. In external backend, only the H2O client running in the Spark driver is affected by spark memory configuration